EIPS/eip-3369.md

```diff
@@ -0,0 +1,183 @@
+---
+eip: 3369
+title: Apply minor modifications to the Ethash algorithm to break current ASIC implementations
+status: Draft
+type: Standards Track
+category: Core
+author: mineruniter969 <mineruniter969@tutanota.com>
+created: 2021-03-13
+---
+
+# Simple Summary
+
+This EIP proposes minor modifications to the current Ethash PoW algorithm to kick ASICs out of the network.
+
+# Abstract
+
+This EIP is to kick current ASIC implementations out of the network to keep the Ethereum network secure and healthy.
+
+# Motivation
+
+ASICs provide a severe centralization risk for the Ethereum network. If we do not get rid of them, small GPU miners will be forced to exit the Ethereum mining because EIP-1559 will make them mining at a loss. Furthermore, ASIC production will be concentrated only at one or two parties, which will make the Ethereum hashrate centralized. Also, it is worth noting that Ethash ASIC machines cost 10k+ USD, while GPUs are priced less than 1000 USD.
+
+With GPUs, miners can switch to other mining algorithms, but with ASICs, it is impossible. Leaving everything as-is will almost certainly a very tough (from the side of miners) integration of the Ethereum 2.0.
+
+In short, this EIP is required to keep the Ethereum network stable and decentralized by keeping the ASIC business away.
+
+# Specification
+
+If `block.number >= ETHASH11_BLKNUM`, activate the `ethash1.1` algorithm version.
+
+## ethash1.1
+
+The changes done are minor yet effective to break ASIC implementations.
+
+Right now, `fnv` hash function is used throughout the `hashimoto` function. `fnv` is identical for all steps, `ethash1.1` will introduce additional `fnvA`, `fnvB`, `fnvC`, `fnvD`, and `fnvE` functions. All those functions will have different FNV constants.
+
+```c
+// Previously used FNV prime
+#define FNV_PRIME_0 0x1000193
+
+// New FNV primes
+#define FNV_PRIME_A 0x10001a7
+#define FNV_PRIME_B 0x10001ab
+#define FNV_PRIME_C 0x10001cf
+#define FNV_PRIME_D 0x10001e3
+#define FNV_PRIME_E 0x10001f9
+```
+
+The FNV prime selection is based on David Stanfill's work on EIP-969.
+
```

Comment:
```suggestion
```
Attribution is not required, and not appropriate for the specification section.  Also, 969 is a DRAFT EIP which means referencing it here will block moving this EIP forward until that EIP is moved forward.  Recommend removing this line.